src/PrestaShopBundle/Routing/Converter/LegacyUrlConverter.php

```diff
@@ -30,7 +30,7 @@
 use PrestaShopBundle\Routing\Converter\Exception\ArgumentException;
 use PrestaShopBundle\Routing\Converter\Exception\RouteNotFoundException;
 use Symfony\Component\HttpFoundation\Request;
-use Symfony\Component\Routing\Exception\Exceptioninterface;
```

Comment:
🤦 

So ... did it work before because php is case-insensitive ?! 😅

Reply:
Surprise